tests/app/sample_data.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
what's the difference between `sample_data` and `sample_batch`? Do you mean batch in the table name sense?

Reply:
this is an obsolete older version.  I'll remove it.  good catch